golos.vesting/golos.vesting.cpp

```diff
@@ -74,6 +74,11 @@ void vesting::on_transfer(name from, name to, asset quantity, std::string memo)
     add_balance(recipient != name() ? recipient : from, converted, has_auth(to) ? to : from);
 }
 
+void vesting::on_bulk_transfer(name from, std::vector<recipient> recipients) {
+    for (auto recipient_obj : recipients)
+        on_transfer(from, recipient_obj.to, recipient_obj.quantity, recipient_obj.memo);
```

Comment:
we can further optimize this if cache `token::get_issuer()`, `table_vesting.find()` and `token::get_balance()` (inside `token_to_vesting`) calls, which result intrinsic calls, which are slow.
caching vesting stats and token balance also makes exchange price stable and removes side-effect of sequential converting (now supply and balance changes between steps, and same `amount` can result slightly different `converted` due rounding).
@s-medvedev, what do you think?